### DISCUSSION TITLE
Migration file in `site-packages` folder for generating a custom API Key model

### DISCUSSION THREAD

**Discussion by hr23232323** [florimondmanca/djangorestframework-api-key#250] [Q&A]
Hello @florimondmanca 

I was following the instructions in the [customization section](https://florimondmanca.github.io/djangorestframework-api-key/guide/#customization) of the user guide docs and getting slightly confused about the **model generating migration(s)**. 

### Additional Details
As far as I can tell, when creating a custom API key model (which inherits from `AbstractAPIKey`) a migration is generated and stored in `site-packages` folder (ex path: `/home/django/.local/lib/python3.10/site-packages/rest_framework_api_key/migrations/0005_auto_20220110_1102.py`). 

This means that no django migration is present in the repository (`site-packages` folder is usually ignored). Is this expected or am I doing something wrong?

### Issue(s)
- When collaborating with other developers/deploying to production, the migration files(s) get regenerated each time instead of being version controlled with other migration file(s) in the repository
- When running Github Action tests on our repo, the migration is missing so we see the following error - `sqlite3.OperationalError: no such table: rest_framework_api_key_<custom api key name>`

### Discussion
- Is this behavior expected?
- To make our GHA tests run, do we need to manually copy over the migration from the `site-packages` folder to the local django migrations folder? Will this cause any issues?

**Comment by florimondmanca** [comment]
Hello @hr23232323,

Seeing this just now as I visit this repo again.

There's a `test_project` in this repository, where a `HeroAPIKey(AbstractAPIKey)` is declared, and _at the time_ the migration was generated in the `migrations` folder of the `heroes` app: https://github.com/florimondmanca/djangorestframework-api-key/blob/master/test_project/heroes/migrations/0001_initial.py

I'm not sure what might be making Django generate migrations in `site-packages` rather than in-project. Is there a Django setting to tweak on your side to fix this?
